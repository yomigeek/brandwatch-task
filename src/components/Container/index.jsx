import styled from 'styled-components';

const Box = styled.div`
  width: 100%;
  background: #fff;
`;

const Cover = styled.div`
//   padding: 20px 30px;
`;

const Container = ({ children }) => {
  return (
    <Cover>
      <Box>{children}</Box>
    </Cover>
  );
};

export default Container;
