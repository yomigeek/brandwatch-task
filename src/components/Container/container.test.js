/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import Container from './index';

test('renders snapshot', () => {
  render(
    <Container>
      <div>Welcome</div>
    </Container>
  );
  const asFragment = render(
    <Container>
      <div>Welcome</div>
    </Container>
  ).asFragment;
  expect(asFragment()).toMatchSnapshot();
});
