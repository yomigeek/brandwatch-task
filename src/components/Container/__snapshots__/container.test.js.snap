// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-gsDKAQ dYrlEF"
  >
    <div
      class="sc-bdvvtL dxscsO"
    >
      <div>
        Welcome
      </div>
    </div>
  </div>
</DocumentFragment>
`;
